pkg/config/agent/config.go

```diff
@@ -158,7 +158,7 @@ type AgentConfig struct {
 	// The minTTL setting helps address the problem of applications caching DNS response IPs indefinitely.
 	// The Cluster administrators should configure this value, ideally setting it to be equal to or greater than the maximum TTL
 	// value of the application's DNS cache.
-	MinTTL int `yaml:"minTTL,omitempty"`
+	FqdnCacheMinTTL int `yaml:"fqdnCacheMinTTL,omitempty"`
```

Comment:
I think the field name here should be `FQDNCacheMinTTL` per our conventions

Reply:
comment not addressed